neuralpp/test/quick_tests/graphical_model/representation/belief_propagation_test.py

```diff
@@ -0,0 +1,37 @@
+from neuralpp.experiments.experimental_inference.belief_propagation import BeliefPropagation
+from neuralpp.inference.graphical_model.representation.factor.pytorch_table_factor import (
+    PyTorchTableFactor,
+)
+from neuralpp.inference.graphical_model.variable.integer_variable import IntegerVariable
+
+
+def test_belief_propagation():
+    cloudy = [0.2, 0.4, 0.4]
+    sprinkler = [0.6, 0.4]
+    rain = [
```

Comment:
bit: you might want to rename this variable to reflect the fact that it is the probability of `rain` given `cloudy`. In fact, you might want to name them `prob_...` to differentiate them from the variables themselves.